backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/weather/WeatherForecastResponse.java

```diff
@@ -0,0 +1,6 @@
+package de.neuefische.ffmjava221.teamprojekt.backend.weather;
+
+import java.util.List;
+
+public record WeatherForecastResponse(List<WeatherData> weather) {
```

Comment:
I dont like the namings of your two record and their attributes.
Maybe you could name this record `DailyWeather` with the List `hourlyWeather`? 🤔

Reply:
Isn't it a problem that the name of the List have to match with the list name in the JSON ?

Reply:
Yeah it is. But you can use  the `JsonProperty ` Annotation to change the name for the serialization/deserialization

Reply:
Ahh cool thank u!!!